tests/unit/test_default_bucket.py

```diff
@@ -26,8 +26,9 @@
 def sagemaker_session():
     boto_mock = Mock(name="boto_session", region_name=REGION)
     boto_mock.client("sts").get_caller_identity.return_value = {"Account": ACCOUNT_ID}
-    ims = sagemaker.Session(boto_session=boto_mock)
-    return ims
+    sagemaker_session = sagemaker.Session(boto_session=boto_mock)
+    sagemaker_session.boto_session.resource("s3").Bucket().creation_date = None
```

Comment:
do you need to add `region_name` to this mock?

Reply:
Doesn't matter in this case. Left it as-is.